src/document/mod.rs

```diff
@@ -84,113 +116,225 @@ impl DocumentEncryptOpts {
 }
 
 impl Default for DocumentEncryptOpts {
-    /// default to only sharing with the creator of the document
+    /// Constructs a `DocumentEncryptOpts` with common values.
+    ///
+    /// The document will have a generated ID and no name. Access to the document will be granted only to its creator.
     fn default() -> Self {
         DocumentEncryptOpts::with_explicit_grants(None, None, true, vec![])
     }
 }
 
 #[async_trait]
 pub trait DocumentOps {
-    /// List all of the documents that the current user is able to decrypt.
+    /// Encrypts the provided document bytes.
+    ///
+    /// This will create an entry for the document in the IronCore service. Returns a `DocumentEncryptResult`
```

Comment:
> This will create an entry for the document in the IronCore service.

Reading this, I'm not entirely sure what it means or what we want to convey here. Maybe we should just remove it entirely? I can't think of a great way to reword it to make it more obvious what we mean. I mostly just don't want people to think that we're storing the encrypted document or anything.

Reply:
The intent of this was to differentiate it from unmanaged encryption, but I agree that it doesn't convey that very well. I think it's important to call out that we're storing _some_ information about their document, but clarify that the data is not stored.

I'll try to work on the wording

Reply:
Did it. Let me know what you think